server/src/auth0/updateUser.graphql

```diff
@@ -0,0 +1,3 @@
+query canUserUpdate($user_id: ID!, $input_id: ID!) {
+  SomeUserExists(filter: { AND: [{ id: $user_id }, { id: $input_id }] })
```

Comment:
What is the `$input_id`?

Reply:
`$input_id ` is the `id` argument of the mutation.